app/views/directives/osc-secrets.html

```diff
@@ -0,0 +1,62 @@
+<ng-form name="secretsForm" class="osc-secrets-form">
+  <div ng-repeat="pickedSecret in pickedSecrets">
+    <div class="form-group">
+      <div class="row picked-secret">
+        <div ng-class="type === 'Source Secret' ? 'col-sm-6' : 'col-sm-12'">
+          <label class="picker-label" ng-show="$index === 0">{{type | startCase}}</label>
+          <ui-select ng-model="pickedSecret.name">
+            <ui-select-match allow-clear="{{type === 'Source Secret' ? false : true}}" placeholder="Secret name">{{$select.selected}}</ui-select-match>
+            <ui-select-choices repeat="secret in (allSecrets | filter : $select.search)">
+              <div ng-bind-html="secret | highlight : $select.search"></div>
+            </ui-select-choices>
+          </ui-select>
+        </div>
+
+        <div class="col-sm-6" ng-if="type === 'Source Secret'">
+          <div class="directory">
+            <label for="{{additionalInputFieldName}}" ng-show="$index === 0">
+              {{additionalInputFieldName | startCase}}
+              <span class="help action-inline">
+                <a href>
+                  <i class="pficon pficon-help" aria-hidden="true"
+                  data-toggle="tooltip" data-original-title="{{additionalInputFieldTooltip}}">
+                </i>
+                </a>
+              </span>
+            </label>
+            <div>
+              <input class="form-control"
+                id="{{additionalInputFieldName}}"
```

Comment:
`additionalInputFieldName` might not be a valid `id`.

I'd like to get rid of it anyway if we can (see my earlier comments).